browser/net/brave_static_redirect_network_delegate_helper.cc

```diff
@@ -74,6 +74,13 @@ int OnBeforeURLRequest_StaticRedirectWorkForGURL(
       URLPattern::SCHEME_HTTP | URLPattern::SCHEME_HTTPS,
       "*://dl.google.com/*");
 
+  static URLPattern widevineGvt1_pattern(
```

Comment:
most of names here don't match CC, I think it should be `widevine_gvt1_pattern`

Reply:
I filed this issue: https://github.com/brave/brave-browser/issues/10314 - Will try to address this soon. Added a to-do to the code.